<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HashtagDialog</class>
 <widget class="QDialog" name="HashtagDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="7">
    <widget class="QScrollArea" name="saHashtags">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="wHashtags">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>923</width>
        <height>522</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_15">
       <item row="1" column="0">
        <layout class="QGridLayout" name="grlHashtags">
         <property name="spacing">
          <number>2</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbSorting">
     <property name="text">
      <string>Порядок</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cbSorting">
     <item>
      <property name="text">
       <string>По алфавиту</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>По популярности</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>По новизне</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="lblMinRank">
     <property name="text">
      <string>Ранг от:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSpinBox" name="sbMinRank"/>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="lblMaxRank">
     <property name="text">
      <string>до:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QSpinBox" name="sbMaxRank"/>
   </item>
   <item row="1" column="6">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HashtagDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HashtagDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
